board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
Again not needed, the problem is in your uboot port, you must set the MODEL variable to the dtb name (minus ".dtb")